<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="accComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="call_account">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>12</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(30, 30, 50);
color: rgb(255, 255, 255);
font: 10pt &quot;맑은고딕&quot;;</string>
    </property>
    <property name="text">
     <string>계좌평가잔고내역 확인</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>459</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="acc_manage">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(30, 30, 50);
color: rgb(255, 255, 255);
font: 10pt &quot;맑은고딕&quot;;</string>
    </property>
    <property name="text">
     <string>계좌 관리</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Danger_wd">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>601</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(150, 150, 150);
font: 10pt &quot;맑은고딕&quot;;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>621</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>선정 종목 확인 및 DB화</string>
    </property>
    <widget class="QPushButton" name="Del_Stock">
     <property name="geometry">
      <rect>
       <x>524</x>
       <y>30</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(30, 30, 50);
color: rgb(255, 255, 255);
font: 10pt &quot;맑은고딕&quot;;</string>
     </property>
     <property name="text">
      <string>DB 삭제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Load_Stock">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(30, 30, 50);
color: rgb(255, 255, 255);
font: 10pt &quot;맑은고딕&quot;;</string>
     </property>
     <property name="text">
      <string>DB 로드</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Save_Stock">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(30, 30, 50);
color: rgb(255, 255, 255);
font: 10pt &quot;맑은고딕&quot;;</string>
     </property>
     <property name="text">
      <string>DB 저장</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="buylast">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>591</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(150, 150, 150);
font: 10pt &quot;맑은고딕&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>120</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>종목 이름 입력</string>
    </property>
    <widget class="QTextEdit" name="searchItemTextEdit2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(30, 50, 30);
color: rgb(255, 255, 255);
font: 10pt &quot;맑은고딕&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="additmelast">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(30, 30, 50);
color: rgb(255, 255, 255);
font: 10pt &quot;맑은고딕&quot;;</string>
     </property>
     <property name="text">
      <string>종목 추가</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Deletcode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(30, 30, 50);
color: rgb(255, 255, 255);
font: 10pt &quot;맑은고딕&quot;;</string>
     </property>
     <property name="text">
      <string>선정 종목 삭제</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>매수 가격</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>176</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>익절 가격</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>227</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>손절 가격</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>116</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>매수 수량</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="buy_price">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="n_o_stock">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>87</width>
       <height>27</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="profit_price">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>192</y>
       <width>86</width>
       <height>26</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="loss_price">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>250</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>385</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>자동매매 시작</string>
    </property>
    <widget class="QPushButton" name="Auto_start">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>자동매매 시작</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>계좌 내역</string>
     </attribute>
     <widget class="QTableWidget" name="stocklistTableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: rgb(150, 150, 150);
font: 10pt &quot;맑은고딕&quot;;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>미체결 잔고</string>
     </attribute>
     <widget class="QTableWidget" name="not_account">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>451</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>560</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>프로그램 업데이트 및 만들기</string>
    </property>
    <widget class="QPushButton" name="program_g">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(255, 255, 255);
color: rgb(200, 20, 20);
font: 90 10pt &quot;맑은 고딕&quot;;
</string>
     </property>
     <property name="text">
      <string>프로그램 강의 블로그</string>
     </property>
    </widget>
    <widget class="QPushButton" name="program_u">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>92</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(255, 255, 255);
color: rgb(200, 20, 20);
font: 90 10pt &quot;맑은 고딕&quot;;
</string>
     </property>
     <property name="text">
      <string>프로그램 강의 유튜브</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stock_pre">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background-color: rgb(255, 255, 255);
color: rgb(200, 20, 20);
font: 90 10pt &quot;맑은 고딕&quot;;
</string>
     </property>
     <property name="text">
      <string>종목추천 블로그</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
